### DISCUSSION TITLE
Is this compatible with Oqtane 5.2?

### DISCUSSION THREAD

**Discussion by EEddyEdwards** [oqtane/Oqtane.Theme.Corporate#4] [Q&A]
I tried using this approach for making a theme for Oqtane 5.2 but the theme files didn't go into the folder where Oqtane 5.2 was looking for them.

**Comment by sbwalker** [comment]
Yes it is compatible. The project relies on a specific folder structure where the Theme project must be a sibling of the Oqtane Framework

```
Root folder
  - Oqtane Framework folder
    - files
  - Oqtane.Theme.Arsha folder
    - files
```

The structure above is how the Package project deploys the assemblies and assets to the appropriate locations. If you want to use a different directory structure then you will need to modify the /Package/debug.cmd command file to ensure the paths are correct.

**Comment by EEddyEdwards** [comment]
Thanks for your answer and all your work on Oqtane.  I have been using Blazor from the beginning at work and plan to use Oqtane for my personal website.
________________________________
